elevate-self-creation-portal/controllers/selfcreationportal.js

```diff
@@ -1,8 +1,160 @@
 const routeConfigs = require('../constants/routes')
 const requesters = require('../utils/requester')
-const requestParser = require('../utils/requestParser')
+const common = require('../constants/common')
+
+const fetchResources = async (req, res, selectedConfig) => {
+	try {
+
+		const selectedConfig = routeConfigs.routes.find((obj) => obj.sourceRoute === req.sourceRoute)
+
+		// project base path and url
+		const projectPath = selectedConfig.targetRoute.paths[0].path
+		const projectBaseUrl = selectedConfig.targetRoute.paths[0].baseUrl
+
+		// survey base path and url
+		const surveyPath = selectedConfig.targetRoute.paths[1].path
```

Comment:
@adithyadinesh0412 this will not be there always

Reply:
made it dynamic